storage/client_test.go

```diff
@@ -156,11 +156,23 @@ func (m *multiTestContext) Start(t *testing.T, numStores int) {
 }
 
 func (m *multiTestContext) Stop() {
-	m.clientStopper.Stop()
+	var stoppers []*util.Stopper
+	stoppers = append(stoppers, m.clientStopper)
 	for _, s := range m.stoppers {
+		stoppers = append(stoppers, s)
```

Comment:
You could replace lines 159-163 with:

    stoppers := append([]*util.Stopper{m.clientStopper}, m.stoppers...)